wiki/Difficulties/osu!/Hard/en.md

```diff
@@ -21,7 +24,7 @@ Difficulty setup should include a BPM appropriate slider velocity that captures
 And try to avoid these:
 
 -   extremely long streams
--   fullscreen jump patterns ([Jumps](Jumps) that are wide as the screen)
+-   fullscreen jump patterns ([Jumps](/wiki/Mapping_Techniques/Jumps) that are wide as the screen)
```

Comment:
(stating again)
`the jumps link is changed to [jumps](/wiki/Mapping_Techniques/Jump)`